achievements/account_pages/config.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+from user_interaction.config import AccountConfig
+from .views import AchievementAccountView
+
+
+class AchievementConfig(AccountConfig):
+    url_keyword = 'achievements'
+    tab_select_keyword = 'tab_achievements'
+    name = 'My achievements'
+    url_name = 'achievements'
+
+    def get_urls(self):
+        """ Builds a list of urls """
+        return [
+            path('', AchievementAccountView.as_view(), name='achievements'),
```

Comment:
There's a bit of an overlap between `tab_select_keyword` and `AchievementAccountView.selected_tab_name`. Perhaps pass it in as a kwarg in as_view so it's in a single place?
```suggestion
            path('', AchievementAccountView.as_view(selected_tab_name=self.tab_select_keyword), name='achievements'),
```
The same goes for the accountconfigs of the other modules.